lib/templates/test-body-footer.html

```diff
@@ -12,7 +12,7 @@
     var data = JSON.stringify(coverageData || {});
 
     var request = new XMLHttpRequest();
-    request.open('POST', '/write-coverage', REQUEST_ASYNC);
+    request.open('POST', '/write-coverage');
```

Comment:
XHR's `.open` method defaults to async if not given a value of `false` for the 3rd argument.

https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/open